content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,362 @@
+have an effect on which products are taken, from where, and when.
 
 .. image:: removal/empty-dock.png
    :align: center
-   :alt: empty stock waiting for deliveries at the docks.
+   :alt: Empty stock waiting for deliveries at the docks.
+
+In this example, vendor trucks unload pallets of goods at the docks. Then, operators scan the
+products in the receiving area, with the reception date and expiration date (if applicable). After
+that, products are stored in their respective locations.
+
+In Odoo, receiving products is done either in the *Inventory* or *Barcode* app. In either app, in
+the default kanban view, click :guilabel:`Receipts`, then, click on the individual receipt. Next, in
+the *Inventory* app, click :guilabel:`Edit`, then enter the received quantity in the
+:guilabel:`Done` column. Finally, click :guilabel:`Validate` to receive the products.
 
-Here, vendor trucks unload pallets of goods at the docks. Then, operators scan the products in the
-receiving area, with the receiving date and, if the product has an expiration date, the expiration
-date. After that, products are stored in their respective locations.
+If using the *Barcode* app, scan the product(s), update the quantity, and finally, click
+:guilabel:`Validate`. After products are received in Odoo, the products can be moved to their
+respective storage locations.
 
 .. image:: removal/entering-stocks.png
    :align: center
-   :alt: products entering stock via the receiving area.
+   :alt: Products entering stock via the receiving area.
 
-Next, several orders for the same product are made, but you didn’t receive the goods the same day
-and they don’t have the same expiration date. In that situation, you logically prefer sending those
-with the closest date first. Depending on the removal strategy you chose, Odoo generates a transfer
-with the products fitting your settings the best.
+Next, several orders for the same product are made, but in this example, the goods weren't received
+on the same day, and they don't have the same expiration date. In that situation, logically, sending
+the products with the closest expiration date is preferred. Depending on the chosen removal
+strategy, Odoo generates a transfer with the products that best fit the settings.
 
 .. image:: removal/packing-products.png
    :align: center
-   :alt: products being packed at packing area for delivery, taking expiration dates into account.
+   :alt: Products being packed at the packing area for delivery, taking the expiration dates into
+         account.
 
 .. note::
-   On the transfer form, you can find the product’s lot/serial number to pick for delivery.
+   To pick for delivery, the product's lot/serial number can be found on the transfer form. To learn
+   more about picking and shipping, refer to either
+   :doc:`../../management/shipments_deliveries/receipts_delivery_two_steps`
+   or :doc:`../../management/shipments_deliveries/delivery_three_steps`.
```

Comment:
Let's move "or" since it is an early line break :)
```suggestion
   :doc:`../../management/shipments_deliveries/receipts_delivery_two_steps` or
   :doc:`../../management/shipments_deliveries/delivery_three_steps`.
```